### PR TITLE
Add an SSGTS option to inject a product to the FIPS certified check

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Add an SSGTS option to inject a product to the FIPS certified check.

#### Rationale:

- SSGTS using Github Actions uses Fedora as a testing environment and installed_OS_is_FIPS_certified is dependency of many rules and to make these rules behave well we needed to inject a check there so rules can pass as expected.

### CODE REVIEW

File: linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs/bash/shared.sh [hunk 1]
```diff
@@ -3,3 +3,5 @@
 {{{ bash_instantiate_variables("sshd_approved_macs") }}}
 
 {{{ bash_replace_or_append('/etc/ssh/sshd_config', '^MACs', "$sshd_approved_macs", '@CCENUM@', '%s %s') }}}
+
+ls
```
Comment (ggbecker) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs/bash/shared.sh:7]: trigger test of sshd_use_approved_macs

**Comment by ggbecker** [comment]
Results of SSGTS from GH actions (the testing commit will have to be dropped before merging it):

```
Run tests/test_rule_in_container.sh --dontclean --logdir logs_bash --remediate-using bash --name ssg_test_suite --datastream ssg-rhel7-ds.xml sshd_use_approved_macs
INFO - The DataStream contains 2 Benchmarks
INFO - 0 - scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml
INFO - 1 - scap_org.open-scap_cref_--__w--content--content--build--ssg-rhel7-pcidss-xccdf-1.2.xml
INFO - Selected Benchmark is 0
INFO - To select a different Benchmark, use --xccdf-id-number option.
INFO - The base image option has been specified, choosing Podman-based test environment.
INFO - Logging into logs_bash/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sshd_use_approved_macs
INFO - Script default_correct_value.pass.sh using profile (all) OK
INFO - Script rhel7_correct_value.pass.sh using profile xccdf_org.ssgproject.content_profile_cis OK
INFO - Script wrong_value.fail.sh using profile (all) OK
Setting console output to log level INFO
```

File: tests/ssg_test_suite/xml_operations.py [hunk 2]
```diff
@@ -184,6 +185,18 @@ def add_platform_to_benchmark(root, cpe_regex):
             benchmark.insert(platform_index, e)
 
 
+def add_product_to_fips_certified(root, product="fedora"):
+    query = OVAL_DEF_QUERY + "/{0}".format(
+        "oval-def:definition[@id='oval:ssg-installed_OS_is_FIPS_certified:def:1']/"
+        "oval-def:criteria")
```
Comment (jan-cerny) [inline tests/ssg_test_suite/xml_operations.py:191]: I think you can just concatenate them without using the format method.

Comment (ggbecker) [reply tests/ssg_test_suite/xml_operations.py:191]: I could. I just kept it consistent with https://github.com/ComplianceAsCode/content/blob/e033559db69f4b4217f2fdbbf39afbb44a094cab/tests/ssg_test_suite/xml_operations.py#L84 aka copy-paste :)

**Review by matusmarhefka — APPROVED** [review]

**Pull request merged**